docs/resources/enterprise_project.md

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "Enterprise Project Management Service (EPS)"
+---
+
+# huaweicloud_enterprise_project
+
+Use this resource to manage an enterprise project within HuaweiCloud
+
+## Example Usage
+
+```hcl
+resource "huaweicloud_enterprise_project" "test" {
+  name        = "test"
+  description = "example project"
+}
+```
+
+## Argument Reference
+
+* `name` - (Optional, String) Specifies the name of the enterprise project.
+  This parameter can contain 1 to 64 characters. Only letters, digits, underscores (_), and hyphens (-) are allowed.
+  The name must be unique in the domain and cannot include any form of the word "default" ("deFaulT", for instance).
+
+* `description` - (Optional, String) Specifies the description of the enterprise project.
+
+* `type` - (Optional, String) Specifies the type of the enterprise project.
+  The valid values are *poc* and *prod*, default to *prod*.
+
+* `enable` - (Optional, Bool) Specifies whether to enable the enterprise project. Default to *true*.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `status` - Indicates the status of an enterprise project.
+  + 1 indicates Enabled.
+  + 2 indicates Disabled.
+
+* `created_at` - Indicates the time (UTC) when the enterprise project was created. Example: 2018-05-18T06:49:06Z
+
+* `updated_at` - Indicates the time (UTC) when the enterprise project was modified. Example: 2018-05-28T02:21:36Z
```

Comment:
Missing `import` and `timeouts` description.

Reply:
fixed